cypress/integration/footer.spec.ts

```diff
@@ -2,38 +2,30 @@
 
 describe('Test Footer', () => {
   beforeEach(() => {
-    cy.clearLocalStorage()
     cy.visit('/')
   })
 
-  it('first column', () => {
-    cy.get('.footer').find('img').should('be.visible')
-    cy.get('.footer')
+  it('has branding', () => {
+    cy.get('[data-cy=footer]').find('img').should('be.visible')
+
+    cy.get('[data-cy=footer]')
```

Comment:
This can reference the data-cy=copyrightnotice tag you added previously

Reply:
done 544c4bb